cfme/fixtures/pytest_selenium.py

```diff
@@ -881,6 +884,15 @@ def force_navigate(page_name, _tries=0, *args, **kwargs):
         kwargs.pop("start", None)
         force_navigate("dashboard")  # Start fresh
 
+    # Same with rails errors
+    rails_e = get_rails_error()
+    if rails_e is not None:
+        logger.warning("Page was blocked by rails error, recycling.")
+        logger.error(rails_e)
+        quit()
+        kwargs.pop("start", None)
+        force_navigate("dashboard")  # Start fresh
```

Comment:
blocker_div seems to work the same way, which makes sense in that case, but I don't think the same technique works as well here.